packages/react-table/src/demos/Table.md

```diff
@@ -2252,6 +2247,388 @@ class FilterTableDemo extends React.Component {
 }
 ```
 
+### Sortable - responsive
+
+```js isFullscreen
+import React from 'react';
+import {
+  Button,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  OptionsMenu,
+  OptionsMenuToggle,
+  Pagination,
+  PaginationVariant,
+  Text,
+  TextContent,
+  Select,
+  SelectVariant,
+  Nav,
+  NavList,
+  NavItem,
+  Dropdown,
+  DropdownItem,
+  DropdownGroup,
+  DropdownToggle,
+  Divider,
+  KebabToggle,
+  Avatar,
+  OptionsMenuItem,
+  OptionsMenuSeparator,
+  OptionsMenuItemGroup,
+  Page,
+  PageSidebar,
+  Masthead,
+  MastheadToggle,
+  PageToggleButton,
+  MastheadMain,
+  MastheadContent,
+  SkipToContent,
+  PageSection
+} from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+import CodeIcon from '@patternfly/react-icons/dist/esm/icons/code-icon';
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import FilterIcon from '@patternfly/react-icons/dist/esm/icons/filter-icon';
+import SortAmountDownIcon from '@patternfly/react-icons/dist/esm/icons/sort-amount-down-icon';
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import AttentionBellIcon from '@patternfly/react-icons/dist/esm/icons/attention-bell-icon';
+
+ComposableTableSortable = () => {
+  const [navActiveItem, setNavActiveItem] = React.useState(0);
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+  const [isKebabDropdownOpen, setIsKebabDropdownOpen] = React.useState(false);
+  const [isFullKebabDropdownOpen, setIsFullKebabDropdownOpen] = React.useState(false);
+
+  const columns = ['Column 1', 'Column 2', 'Column 3', 'Column 4', 'Column 5'];
+  const [rows, setRows] = React.useState([
+    ['one', 'two', 'a', 'four', 'five'],
```

Comment:
I would make the values for column 2 four and five unique, so they can be sorted on.